scheduler/cloudformation/cloudformation.go

```diff
@@ -208,6 +208,10 @@ func (s *Scheduler) submit(ctx context.Context, tx *sql.Tx, app *scheduler.App,
 		return err
 	}
 
+	if err := scheduler.Publish(ss, fmt.Sprintf("Created cloudformation template: %v", *t.URL)); err != nil {
```

Comment:
Part of me feels like we should just ignore errors from `Publish`. If for example, the `emp client` disconnected from the network, this would result in `err == io.EOF` and would cause the deployment to fail.